src/parcsr_ls/par_mgr_setup.c

```diff
@@ -780,18 +826,51 @@ hypre_MGRSetup( void               *mgr_vdata,
    /* loop over levels of coarsening */
    for (lev = 0; lev < num_coarsening_levs; lev++)
    {
+      // wall_time_lev = time_getWallclockSeconds();
       /* check if this is the last level */
       last_level = ((lev == num_coarsening_levs - 1));
 
       /* initialize A_array */
       A_array[lev] = RAP_ptr;
       nloc = hypre_CSRMatrixNumRows(hypre_ParCSRMatrixDiag(A_array[lev]));
 
+      wall_time = time_getWallclockSeconds();
```

Comment:
Should we put this under a `#ifdef SOME_VARIABLE`?

Reply:
`#ifdef HYPRE_PROFILE`?